app/validators/answers/number_answer_validator.py

```diff
@@ -29,6 +35,12 @@ def __init__(self, schema_element, questionnaire_schema):
     def validate(self):
         super().validate()
 
+        self.validate_min_max_is_number()
+
+        # Prevent other validation methods that requires calculations running into errors due to types
+        if len(self.errors) > 0:
```

Comment:
You don't actually need the `len`, since `self.errors` is a list you can just do 
```
if self.errors:
    return self.errors
```
its truthy if the list contains items, and falsy otherwise